config.json

```diff
@@ -927,7 +927,7 @@
         "name": "Transpose",
         "uuid": "9c140fb7-cc8b-411b-b613-a0e0081a9c3f",
         "practices": [],
-        "prerequisites": [],
+        "prerequisites": ["arrays", "for-loops", "strings", "conditionals"],
```

Comment:
Here is my solution : https://exercism.org/tracks/javascript/exercises/transpose/solutions/TomPradat

I'm kinda facing a dilemma here : Either we consider that we can solve this exercise like I did, i.e without `array-transformations` and/or `array-analysis` or we add the two latest prerequisites (as used in the majority of community solutions I've seen) but it's a shame given the difficulty of the exercise.

Either ways, I'd recommend increasing the difficulty to 3 or 4. This exercise resembles matrix whose difficulty is 3.

Reply:
Lets increase the difficulty and lets NOT add the array prereqs. If it's harder, people are bound to have more experience, so we'll see more people solving it your way.

3 is fine.

Reply:
[118baee](https://github.com/exercism/javascript/pull/1974/commits/118baee78de925ffc25fc0c349ccec8898368dce)